demo/static/css/main.css

```diff
@@ -2,11 +2,11 @@
    Capital Framework
    Core Less file
    ========================================================================== */
-/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```

Comment:
This looks autogenerated, but—weird—why's the version going down and the URL broken?